src/Cafe/OS/libs/nsyshid/nsyshid.cpp

```diff
@@ -256,6 +258,17 @@ namespace nsyshid
 						 device->m_productId);
 	}
 
+	void AttachEmulatedDevices()
+	{
+		if (GetConfig().emulated_usb_devices.emulate_skylander_portal && !backendList.front()->m_foundSkylander)
+		{
+			cemuLog_logDebug(LogType::Force, "Attaching Emulated Portal");
+			// Add Skylander Portal
+			auto device = std::make_shared<SkylanderPortalDevice>();
+			AttachDevice(device);
+		}
```

Comment:
How about moving this code to a new backend, attaching that backend in `AttachDefaultBackends`, and running the code in `AttachVisibleDevices`?
This would require a method in the `Backend` class to check for devices that were attached by other backends though; see my other comment [here](https://github.com/cemu-project/Cemu/pull/971#discussion_r1331513601).
That way, there wouldn't be any device specific code in nsyshid.cpp, like what is currently the case for libusb and Windows HID stuff.